### ISSUE TITLE
Revise definition and comment of Overvalued idea (HP:5200402)

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#10186]
**HPO term**
Overvalued idea

**Note**
The current comment and definition are below. Please comment with revisions.

**Definition**
A solitary belief that can be considered justified in the context of a person's life history and personality, but which goes on to determine an individual's actions to a morbid degree. The disturbance in functioning and the subjective suffering caused to the patient or others give this type of idea the qualifier of being overvalued.

**Comment**
As opposed to delusions, which are false ideas unique to their possessor, overvalued ideas develop from assumptions and beliefs that are shared by other members of the same culture.

**Comment by giovfico** [comment]
An unreasonable and an isolated sustained belief that is maintained with less than delusional intensity (i.e. the person is able is to acknowledge the possibility that the belief may or may not be true). The belief is not of one that is ordinarily accepted by other members of the person's culture or sub-culture.


An overvalued idea is held strongly, with less than delusional intensity, usually preoccupies the individual's mental life, compared to many delusions, is ego-syntonic, the content is usually regarded as abnormal compared to the general population (but not bizarre as some delusions), could progress to a delusion and may be associated with a high degree of affect (e.g. anxiety or anger when there is a threat to the loss of their goal or object of the belief).

[https://doi.org/10.1016/S0005-7967(01)00016-X]

**Comment by pnrobinson** [comment]
Already in HPO
[Overvalued idea HP:5200402
](https://hpo.jax.org/app/browse/term/HP:5200402). TO DO revise definition

**Comment by pnrobinson** [comment]
updated in hp-edit.owl
